src/_gettsim_tests/test_data/arbeitsl_geld_2.csv

```diff
@@ -35,12 +35,12 @@ hh_id,tu_id,p_id,jahr,kind,alter,rentner,in_ausbildung,geburtsjahr,bürgerg_bezu
 15,15,37,2013,FALSE,40,FALSE,FALSE,1973,FALSE,400,80,70,TRUE,FALSE,800,20,0,0,0,100,0,0,0,0,160,0,0,0,14000,0,900,240,500,0,690,480,0,0,0,,6.857142857,70,2,,Test with wealth (calculated wealth allowance with: https://www.steuerschroeder.de/Steuerrechner/Schonverm%C3%B6gen.html),
 15,15,38,2013,FALSE,40,FALSE,FALSE,1973,FALSE,400,80,70,TRUE,FALSE,0,0,0,0,0,0,0,0,0,0,0,0,0,0,14000,0,0,0,0,0,690,480,0,0,0,,6.857142857,70,2,,Test with wealth (calculated wealth allowance with: https://www.steuerschroeder.de/Steuerrechner/Schonverm%C3%B6gen.html),
 16,16,39,2023,FALSE,45,FALSE,FALSE,1978,FALSE,400,50,60,FALSE,FALSE,1200,20,0,0,0,0,0,0,0,0,204,0,0,0,40000,0,1200,348,648,0,502,450,0,304,304,,10.83333333,60,1,,https://buergergeld-rechner.org/ (Test uses the wrong Regelbedarf for one adult (520 instead of 502) thus result is adjusted by that difference),
-17,17,40,2023,FALSE,40,FALSE,FALSE,1983,FALSE,650,100,70,FALSE,FALSE,0,0,0,0,0,0,0,0,0,0,0,219,0,0,85000,0,0,0,0,0,1220,750,0,0,0,,,,,,https://buergergeld-rechner.org/,
-17,17,41,2023,FALSE,38,FALSE,FALSE,1985,FALSE,650,100,70,FALSE,FALSE,700,12,0,0,0,0,0,0,0,0,140,219,0,0,85000,0,700,238,322,0,1220,750,0,0,0,,,,,,https://buergergeld-rechner.org/,
-17,17,42,2023,TRUE,3,FALSE,FALSE,2020,FALSE,650,100,70,FALSE,FALSE,0,0,0,0,0,0,0,0,0,0,0,219,0,0,85000,0,0,0,0,0,1220,750,0,0,0,,,,,,https://buergergeld-rechner.org/,
-18,18,43,2023,FALSE,40,FALSE,FALSE,1983,FALSE,650,100,70,FALSE,FALSE,0,0,0,0,0,0,0,0,0,0,0,219,0,0,50000,0,0,0,0,0,1220,750,0,1429,1429,,,,,,https://buergergeld-rechner.org/,
-18,18,44,2023,FALSE,38,FALSE,FALSE,1985,FALSE,650,100,70,FALSE,FALSE,700,12,0,0,0,0,0,0,0,0,140,219,0,0,50000,0,700,238,322,0,1220,750,0,1429,1429,,,,,,https://buergergeld-rechner.org/,
-18,18,45,2023,TRUE,3,FALSE,FALSE,2020,FALSE,650,100,70,FALSE,FALSE,0,0,0,0,0,0,0,0,0,0,0,219,0,0,50000,0,0,0,0,0,1220,750,0,1429,1429,,,,,,https://buergergeld-rechner.org/,
-19,19,46,2023,FALSE,30,FALSE,FALSE,1993,TRUE,720,110,105,FALSE,FALSE,1200,20,0,0,0,0,0,0,0,0,204,0,0,0,0,0,1200,348,648,0,1220,592.86,0,842.86,842.86,,7.904761905,75,3,,Constructed Test Case after Karenzzeit with cost for living from testcase in 2011,
-19,19,47,2023,TRUE,3,FALSE,FALSE,2020,TRUE,720,110,105,FALSE,FALSE,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1220,592.86,0,842.86,842.86,,7.904761905,75,3,,Constructed Test Case after Karenzzeit with cost for living from testcase in 2011,
-19,19,48,2023,FALSE,35,FALSE,FALSE,1988,TRUE,720,110,105,FALSE,FALSE,700,20,0,0,0,0,0,0,0,0,140,0,0,0,0,0,700,238,322,0,1220,592.86,0,842.86,842.86,,7.904761905,75,3,,Constructed Test Case after Karenzzeit with cost for living from testcase in 2011,
+17,17,40,2023,FALSE,40,FALSE,FALSE,1983,FALSE,650,100,70,FALSE,FALSE,0,0,0,0,0,0,0,0,0,0,0,219,0,0,85000,0,0,0,0,0,1240,750,0,0,0,,,,,,https://buergergeld-rechner.org/ (considering Sofortzuschlag),
```

Comment:
So, does not https://buergergeld-rechner.org consider Sofortzuschlag?

Then I would write something like (adjusted for Kindersofortzuschlag)